### ISSUE TITLE
wrapping up the manuscript

### ISSUE THREAD

**Issue by HWRix** [mkness/TheCannon#42]
**two batches of comments from Anna**

=================================
a couple of general comments, referring to the outline below. 

For the most part, sections 1-3 were very clear!
The only exception: I found 2b confusing. I think a couple of sentences need to be reshuffled, see annotations.
I found it strange to use the words "approximations" and "limitations" interchangeably at the beginning of 4. And in particular, I had a hard time following 4a, the passage on "issues with labels." 

Structure of discussion:
    1) Overview of what The Cannon is, how it works
    2) Overview of APOGEE DR10 application
    3) Practical advantages
    a. Speed, explain why
    b. Same precision at much lower SN, explain why
    c. Provides robust continuum estimate
    4) Approximations and limitations
    a. Issues with labels (I find this section confusingly organized)
    i. Labels are in reality noisy and biased *and* we're not taking advantage of information about the unlabeled stars
    ii. No set of reference objects is all-encompassing
    iii. Label space dimensionality is always incomplete
    iv. The approach to solving (1) and (3) and what we did to deal with systematic errors in the reference labels? Confusing
    v. Issues (2) and (3) are linked in that both can be addressed by expanding label space ? Confusing
    b. Limiting assumptions about the spectral model pixel-by-pixel Ansatz
    5) Question of suitable reference objects
    6) The potential for cross-survey calibration, bringing different surveys onto a consistent stellar parameter and chemical abundance scale

===================================
 here are some questions & comments from the rest of the doc:

Do we want to say only "stellar parameter" in the title when in the rest of the paper we make a to-do about our labels being parameters + abundances? 
I think we need to insert a sentence in the abstract that explicitly says, "The Cannon is..." At present, we just jump right into describing it! 
Section (2) is still confusing to me, and I think it's because of the order in which general vs. APOGEE-specific ideas are presented. I know that the idea is for 2.1 and 2.2 to be more general, and for 2.3 to be the APOGEE illustration, but right now 2.1 and 2.2 both have an APOGEE illustration within them anyway. So, it doesn't make sense for 2.3 to be the introduction to characteristics of the APOGEE dataset. I also think it's confusing for 2.3.1 to be a subset of 2.3. This order makes a lot more sense to me personally:

2. Data
    2.1 Specifics of the APOGEE Data Set 
    2.2 Choosing Reference Objects for the Training Step
    2.3 Consistent Continuum Normalization
    2.4 Labels for the Reference Objects in APOGEE

    Since we call section 4 "The Cannon's Test Step: Labeling Survey Spectra" I think we should call section 3 "The Cannon's Training Step: Making a Generative Model" or something like that, to be clear that these are the two fundamental steps of our method
    I think it's strange to call Section 5 "The Cannon in Action", since really Sections 3 & 4 also show The Cannon in Action. Can we just say "Results with APOGEE Data" or "The Cannon Illustrated with APOGEE Data"?
    aspcapstar or aspcapStar?
    ASPCAP or ASPCAP?
    APOGEE or APOGEE?
    (For the above two, being particularly careful in section titles)
    Italicization of our own terms is inconsistent. I think we should italicize the first time, and then not again (except for The Cannon). Currently, for example, it's "reference objects" the first time (in the abstract!), "reference objects" the second time in the abstract, "reference objects" the next three times, "reference objects" once, "reference objects" again, and then finally "reference objects" for the rest of the paper. 
    Should be consistent in our capitalization of section headings. For example, right now we have 2.1 "Choosing reference objects" followed immediately by 2.2 "Consistent Continuum Normalization"
    We use the term "signal-to-noise" a zillion times before finally saying "(hereafter "SNR")" all the way on page 7. Not a big deal, but I found it odd.

===================================
